src/db.ts

```diff
@@ -0,0 +1,42 @@
+import { Sequelize, SequelizeOptions } from "sequelize-typescript";
```

Comment:
If this is not part or used by the exported libraries, please move it to a different folder to make clear that this is only used for tests. (make a testUtils folder or something like that).